### PR TITLE
feat(db): migration - add name to movies

### PR DESCRIPTION
**Pull request by SishaarRao**
**Why:** We would like to rename a column with 0 downtime

**What:** First migration step in order to rename the column title to name

### CODE REVIEW

**Comment by SishaarRao** [comment]
Right now, this branch is set to merge into controller since the controller PR hasn't been merged yet. Once the controller branch is merged, I'll fast forward this branch and then merge.

File: db/migrations/20200226203241_add_name_to_movies.js
Comment (Lianathanoj) [inline db/migrations/20200226203241_add_name_to_movies.js:6]: at this point, because most of our node repos are node 10/12+, can you trying using `async await` here too? it's just a good practice going forwards whenever possible.

**Comment by Lianathanoj** [comment]
what was your reasoning for using `fix` in the PR title?

**Review by Lianathanoj — APPROVED** [review]
LGTM pending merge of other PR + rebase of this PR afterwards as well as nits (i.e. potentially changing your PR title and using async/await).

File: db/migrations/20200226203241_add_name_to_movies.js
Comment (karankwatra) [reply db/migrations/20200226203241_add_name_to_movies.js:6]: +1

**Review by karankwatra — APPROVED** [review]
LGTM but +1 to jb's comment about using async/await! Let me know if you have questions about how to refactor that

**Pull request merged**
